### PR TITLE
Mobile subscriptions

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

File: app/src/page/subscriptions/view.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  FlatList,
+  NativeModules,
+  SectionList,
+  Text,
+  View
+} from 'react-native';
+import { buildURI } from 'lbry-redux';
+import { uriFromFileInfo } from '../../utils/helper';
+import moment from 'moment';
+import Colors from '../../styles/colors';
```
[the remark this answers is not in the document]
Comment (akinwale) [reply app/src/page/subscriptions/view.js]: Finally added module-resolver to `.babelrc` to sort this out..

File: app/src/index.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ import settingsReducer from './redux/reducers/settings';
 import thunk from 'redux-thunk';
 
 const globalExceptionHandler = (error, isFatal) => {
+  console.log(error);
```
[the remark this answers is not in the document]
Comment (akinwale) [reply app/src/index.js]: Yes.

**Comment by tzarebczan** [comment]
On first load, it took some time for the subs to show up. Can we add a loading icon? 

Should we have 2 different views similar to the desktop app? Right now it looks like it's by creator, then date..so I don't see the latest content on top.

File: app/package.json [hunk 3]
```diff
@@ -7,8 +7,9 @@
     },
     "dependencies": {
         "base-64": "^0.1.0",
+        "@expo/vector-icons": "^8.1.0",
```
Comment (skhameneh) [inline app/package.json]: How does this compare to [react-native-vector-icons](https://www.npmjs.com/package/react-native-vector-icons)?

File: app/src/component/AppNavigator.js [hunk 4]
```diff
@@ -148,6 +160,9 @@ const drawer = createDrawerNavigator({
   TrendingStack: { screen: trendingStack, navigationOptions: {
     title: 'Trending', drawerIcon: ({ tintColor }) => <Icon name="fire" size={20} style={{ color: tintColor }} />
   }},
+  MySubscriptionsStack: { screen: mySubscriptionsStack, navigationOptions: {
```
Comment (skhameneh) [inline app/src/component/AppNavigator.js]: Line breaks before and check indentation above

File: app/src/component/button/view.js [hunk 5]
```diff
@@ -41,9 +43,14 @@ export default class Button extends React.PureComponent {
       textStyles.push(buttonStyle.textLight);
     }
 
+    let renderIcon = (<Icon name={icon} size={18} color={iconColor ? iconColor : ('light' === theme ? Colors.DarkGrey : Colors.White)} />);
```
Comment (skhameneh) [inline app/src/component/button/view.js]:
```
const renderIcon = (
  <Icon name={icon} size={18} color={iconColor ? iconColor : ('light' === theme ? Colors.DarkGrey : Colors.White)} solid={!!solid} />
);
```

**Comment by skhameneh** [comment]
Assign back if you need new changes reviewed

**Comment by akinwale** [comment]
@tzarebczan We can do that for a future release. Although, I'd like to make the default order date published so that the latest content shows up at the top.

**Pull request merged**